src/http/server.ts

```diff
@@ -115,6 +120,10 @@ export class Server {
         patterns,
       });
     });
+
+    if (options.error_handler) {
+      this.#error_handler = new options.error_handler();
+    }
```

Comment:
No need for condition. Always apply the default if none exist.

```ts
this.#error_handler = new (options.error_handler() || Drash.ErrorHandler);
```

Reply:
https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgBIQDYYPbIN4BQyyAFpjgBQCUAXMgG7bAAmA3AQL4EEIZwDO-ZADFsuYAFsADhggSI4IeizZCxKQFcARhmAJS5bNXxFiyBNhD9ssgHQ4A5hQBEMMc6qmuXHn0HIAITgoZEkZOQUwJUMTdW1dfTIVYzUzCysbCHtsJ2ctYI8vTm4YDRAEMGBLZC0NYAxmaAoEAH46EAgAd2QKWz7gh346OBAATwBtAF0qZABeAD40QxnU9P4wBjnkDu7m5AAffcDgqnZielskylPiglr6xqhqdnuGptFsKiA